sdk/src/settings/mod.rs

```diff
@@ -426,6 +430,12 @@ impl Settings {
         }
     }
 
+    /// Set the [Settings] from a toml file.
+    pub fn from_json(toml: &str) -> Result<()> {
+        #[allow(deprecated)]
+        Settings::from_string(toml, "json").map(|_| ())
```

Comment:
Maybe it's time to un-deprecate that one `Settings::from_string`?

Reply:
I'm going to un-deprecate it and we can continue to use it. But we should stick with just toml and json.